build/Version.props

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup Condition=" '$(PackageVersion)' == '' ">
 
     <!-- If $(PB_VersionStamp) is empty and $(PB_IsStable) is not true, then a default PackageBuildQuality will be added. -->
-    <PackageReleaseVersion Condition="'$(PackageReleaseVersion)' == '' ">2.1.400</PackageReleaseVersion>
+    <PackageReleaseVersion Condition="'$(PackageReleaseVersion)' == '' ">2.1.402</PackageReleaseVersion>
```

Comment:
cc @vijayrkn - is this the right branding?

Reply:
Yes, this is correct for this branch.